frontend/packages/cli/src/cli/initCommand/index.ts

```diff
@@ -20,7 +20,7 @@ const formatMap: Record<string, string> = {
   PostgreSQL: 'postgres',
   'Ruby on Rails (schema.rb)': 'schemarb',
   'Prisma (schema.prisma)': 'prisma',
-  Drizzle: 'postgres', // Drizzle also uses --format postgres
+  'Drizzle (schema.ts)': 'drizzle',
```

Comment:
I'm concerned about the processing around `handleDrizzlePrompts()`, so I'd like to consider whether to fix it together in this PR, or once it's in postgres, in a separate PR!

Reply:
I wrote the experimental notation. Thank you!
https://github.com/liam-hq/liam/pull/2299/commits/cba9b2f302530bd371369426b765cac05f81fed8

Reply:
I have updated ``handleDrizzlePrompts ``.
https://github.com/liam-hq/liam/pull/2299/commits/4bc1928ff175fa42b9db4cc0a31ad26e3c4bb501

and also changed ``changeset``!